.gitlab/notify/notify.yml

```diff
@@ -104,7 +103,6 @@ notify_gitlab_ci_changes:
   script:
     - python3 -m pip install -r tasks/libs/requirements-github.txt
     - !reference [.setup_agent_github_app]
-    - GITLAB_TOKEN=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $GITLAB_FULL_API_TOKEN) || exit $?; export GITLAB_TOKEN
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why don't we need this anymore?

Reply:
This should have been done with https://github.com/DataDog/datadog-agent/pull/29686. This token was used to obtain the full gitlab ci which is now obtained via artifacts of the `compute_gitlab_ci_config` job

Reply:
Okk nice !